arcgis-ios-sdk-samples/Geometry/Project/README.md

```diff
@@ -0,0 +1,31 @@
+# Project
+
+Project a point to another spatial reference.
+
+## Use case
+
+Being able to project between spatial references is fundamental to a GIS. An example of when you would need to re-project data is if you had data in two different spatial references, but wanted to perform an intersect analysis with the `class AGSGeometryEngine.intersection(ofGeometry1:geometry2:)` method. This method takes two geometries as parameters, and both geometries must be in the same spatial reference. If they are not, you could first use `class AGSGeometryEngine.projectGeometry(_:to:)` to convert the geometries so they match.
+
+![Project App](project.png)
+
+## How to use the sample
+
+Tap anywhere on the map. A callout will display the tapped location's coordinate in the original (basemap's) spatial reference (wgs84) and in the projected spatial reference.
```

Comment:
Original spatial reference is `webMercator` and the projected spatial reference is `wgs84`

Reply:
For clarity, should I replace that portion of the sentence with "spatial reference `webMercator` and in the projected spatial reference `wgs84` ?"

Reply:
Sounds good to me